services/src/main/java/io/druid/cli/InsertSegment.java

```diff
@@ -71,6 +74,9 @@ public InsertSegment()
   protected List<? extends Module> getModules()
   {
     return ImmutableList.<Module>of(
+        new DruidProcessingModule(),
```

Comment:
Sounds good. Perhaps include a comment that we're not sure if these are actually needed or not.